tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -714,22 +714,22 @@
     <string name="proxy_server_user_name">Username:</string>
     <string name="proxy_server_user_pass">Password:</string>
     <string name="proxy_server_empty_field_hint">This field can be left blank</string>
-    <string name="proxy_exclude_apps_from_proxy">Exclude Applications</string>
-    <string name="proxy_exclude_apps_from_proxy_summ">Applications list to open directly. These applications will not use Tor or Proxy to connect to the Internet!</string>
-    <string name="proxy_select_proxy_app"> Please select proxy owner application at least, if you are using a local proxy.</string>
+    <string name="proxy_exclude_apps_from_proxy">Excluded Apps</string>
+    <string name="proxy_exclude_apps_from_proxy_summ">List of apps to open directly. These apps will not use Tor or a proxy to connect to the Internet!</string>
+    <string name="proxy_select_proxy_app"> Please select at least a proxy owner app, if you are using a local proxy.</string>
     <string name="proxy_use_for_dnscrypt">Use proxy for DNSCrypt</string>
     <string name="proxy_use_for_tor">Use proxy for Tor</string>
     <string name="proxy_use_for_itpd">Use proxy for Purple I2P</string>
-    <string name="proxy_save">Save and enable</string>
+    <string name="proxy_save">Save and use</string>
     <string name="proxy_no_connection">Cannot connect to proxy: %s</string>
-    <string name="proxy_successful_connection">Successful connection. Ping %s ms.</string>
+    <string name="proxy_successful_connection">Connected. %s ms ping time.</string>
     <string name="notification_arp_spoofing">ARP spoofing detected!</string>
     <string name="notification_rogue_dhcp">Rogue DHCP detected!</string>
-    <string name="notification_mitm">Man-in-the-middle attack detected! Your data can be intercepted by another device on the local network.
-        Turn off, wait a few seconds, and turn on Wi-Fi. Using current Wi-Fi hotspot may be unsafe!</string>
-    <string name="toast_arp_detection_not_supported">ARP spoofing detection is not supported for your device!</string>
+    <string name="notification_mitm">Man-in-the-middle attack detected! Another device on the local network could be intercepting your data.
+        Turn off, wait a few seconds, and turn on Wi-Fi. Using the current Wi-Fi hotspot may be unsafe!</string>
```

Comment:
```suggestion
        Turn off Wi-Fi, wait a few seconds, and turn it on again. Using the current Wi-Fi hotspot may be unsafe!</string>
```
Wouldn't this usually auto-connect to the same network? Add something about removing the setup?